packages/terra-date-picker/src/terra-dev-site/test/date-picker/calendar-filter/CalendarFilterFilterDates.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import moment from 'moment';
+import CalendarFilter from '../../../../CalendarFilter';
+
+const isWeekdayOrToday = (date) => {
+  const momentDate = moment(date);
+
+  // Ensure if today is a weekend that the Today button doesn't get disabled and causes the screenshot to mismatch.
+  if (momentDate.isSame(moment(), 'day')) {
+    return true;
+  }
```

Comment:
Is this something consumers would need to implement within their filterData function?

Reply:
yes filterDate is a callback function which can be used to filter days in calendar. A return value of true will enable a day and false will disable a day.